spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    first_name "Aaron"
+    last_name "Sumner"
+    email "tester@example.com"
```

Comment:
emailカラムにはunique制約をつけてるので、複数回factoryのcreateが呼び出されると2回目以降は重複エラーが出ちゃう
```suggestion
    sequence(:email) { |n| "tester#{n}@example.com" }
```
↑こういう風に書くと`tester1@example.com`、`tester2@example.com`って感じで違う文字列で保存してくれる